pos_save_client_error_fix/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+POS Save Client Error Message FIX
+==============
+
+This module extends the functionality of POS and displays the error message returned by the server rather than "Your Internet connection is probably down.".
+
```

Comment:
@rafaelbn, how did you tested the module ? If you loose the connection, this message is normal. The module is changing message if connection is OK, but customer creation failed, as far as I understand.